deployment/prometheus/02-prometheus-configmap.yaml

```diff
@@ -297,6 +297,42 @@ data:
         action: replace
         target_label: kubernetes_pod_name
 
+    - job_name: 'envoy-statsd'
+      scrape_interval: 30s
+
+      kubernetes_sd_configs:
+      - role: pod
+
+      relabel_configs:
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+        action: keep
+        regex: true
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_statsdpath]
+        action: replace
+        target_label: __metrics_path__
+        regex: (.+)
+      # The `prometheus.io/format` annotation is required for scraping Envoy
+      # metrics. This is because Envoy does not expose prometheus metrics
+      # without setting the format=prometheus URL parameter on Envoy's stats
+      # enpdoint.
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_format]
```

Comment:
I think we should be able to get rid of this annotation for statsd.

Reply:
We need the `prometheus.io/format` annotation for statsd as well?

Reply:
Ahh I see, sorry misread your comment. Yeah I can ditch that one, it's redundant.